client/common.go

```diff
@@ -199,21 +199,24 @@ func (x *contextCall) processResponse() bool {
 	// get result status
 	st := apistatus.FromStatusV2(x.resp.GetMetaHeader().GetStatus())
 
-	successfulStatus := apistatus.IsSuccessful(st)
-	x.err = apistatus.ErrFromStatus(st)
+	var isErrorExists bool
```

Comment:
```suggestion
	var errorExists bool
```
? otherwise, i dont get it

Reply:
renamed